scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -918,8 +1000,8 @@ public void onSubmitComplete(SubmitCompleteEvent event) {
                             String jsonError = event.getResults().substring(SubmitEditServlet.ERROR.length());
                             JSONValue json = controller.parseJSON(jsonError);
                             JSONObject obj = json.isObject();
-                            if (obj != null && obj.containsKey("errorMessage")) {
-                                String errorMessage = obj.get("errorMessage").toString();
+                            if (obj != null && obj.containsKey(ERROR_MESSAGE)) {
+                                String errorMessage = obj.get(ERROR_MESSAGE).toString();
                                 if (errorMessage.contains("JobValidationException")) {
                                     errorMessage = errorMessage.substring(errorMessage.indexOf(":") + 2);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Put single-quotes around ':' to use the faster "indexOf(char)" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3027)